api/libproject/devstart.rb

```diff
@@ -1217,6 +1217,45 @@ def export_workspace_data(cmd_name, *args)
     :fn => ->(*args) {export_workspace_data("export-workspace-data", *args)}
 })
 
+def delete_workspaces(cmd_name, *args)
+  common = Common.new
+  ensure_docker cmd_name, args
+
+  op = WbOptionsParser.new(cmd_name, args)
+  op.opts.project = TEST_PROJECT
+
+  op.add_typed_option(
+      "--delete-list-filename [delete-list-filename]",
+      String,
+      ->(opts, v) { opts.deleteListFilename = v},
+      "File containing list of workspaces to delete.
+      Each line should contain a single workspace's namespace and firecloud name, separated by a comma
+      Example: ws-namespace-1,fc-id-1 \n ws-namespace-2,fc-id-2 \n ws-namespace-3, fc-id-3")
+
```

Comment:
Could you add a dry-run option that verifies the file list and logs whether each workspace exists or not?

Reply:
good idea